lib/modules/actions.js

```diff
@@ -8,6 +8,8 @@ export const SET_SHELL = 'PLATFORM__SET_SHELL';
 export const REDIRECT = 'PLATFORM__REDIRECT';
 export const REROUTE_PAGE = 'PLATFORM__REROUTE_PAGE'; // re-run the handlers for the page
 
+export const NO_REFERRER  = 'NO_REFERRER';
```

Comment:
ah! that is the place that makes sense for it. It doesn't look like its working though, all my referrers are empty